packages/subgraph/tests/utils/events.ts

```diff
@@ -1,12 +1,30 @@
-// TODO: Remove this file and import from OSx-commons-subgraph,
-// once the OSx-commons-subgraph npm package is published
 import {
   InstallationPrepared,
   InstallationPreparedPreparedSetupDataStruct,
-} from '../generated/PluginSetupProcessor/PluginSetupProcessor';
-import {Address, Bytes, ethereum} from '@graphprotocol/graph-ts';
+} from '../../generated/PluginSetupProcessor/PluginSetupProcessor';
+import {NumberStored} from '../../generated/templates/Plugin/Plugin';
+import {Address, BigInt, Bytes, ethereum} from '@graphprotocol/graph-ts';
 import {newMockEvent} from 'matchstick-as';
 
+export function createNewNumberStoredEvent(
+  number: string,
+  contractAddress: string
+): NumberStored {
+  let createNumberStoredEvent = changetype<NumberStored>(newMockEvent());
+
+  createNumberStoredEvent.address = Address.fromString(contractAddress);
+  createNumberStoredEvent.parameters = [];
+
+  let proposalIdParam = new ethereum.EventParam(
+    'number',
+    ethereum.Value.fromSignedBigInt(BigInt.fromString(number))
+  );
+
+  createNumberStoredEvent.parameters.push(proposalIdParam);
+
+  return createNumberStoredEvent;
+}
```

Comment:
I am not sure it is a good idea to mix different kind of events, it will make the file unmanagebale IMO
I would be doing this another way:
i would move create event related to the plugin to tests/plugin
and move create events related to osx to tests/osx
or i find another way to segregate a bit
WDYT?